src/js/components/SkipLinks/SkipLinks.js

```diff
@@ -35,29 +48,40 @@ const SkipLinks = ({ children, id, messages }) => {
   return (
     <Layer
       id={id}
-      position={showLayer ? 'top' : 'hidden'}
+      position={showLayer ? theme.skipLinks.position : 'hidden'}
       ref={layerRef}
       onFocus={onFocus}
       onBlur={onBlur}
+      tabIndex={0}
+      modal={false}
+      targetChildPosition="first" // prepend the Layer on the body container
+      // Non-modal Layer's will take the full screen at small breakpoints
+      // by default, which isn't what we want, hence setting responsive false
+      responsive={false}
     >
-      <Box pad={{ horizontal: 'medium' }}>
-        <Heading level={2}>{messages.skipTo}:</Heading>
-        <Box direction="row" align="center" pad={{ bottom: 'medium' }}>
-          {children.map((element, index) =>
-            cloneElement(element, {
-              key: `skip-link-${index}`,
-              onClick: removeLayer,
-            }),
+      {showLayer && (
```

Comment:
I would have thought we'd always render each `SkipLink` and let the user's tab take care of giving it focus and causing the Layer to remove 'hidden'.

Reply:
I suspect there might be an issue with how Layer is interpolating `position="hidden"` with `modal={false}` that drove this condition.
Even when the Layer is set to the hidden position, it's still being displayed, though removing modal={false} makes the Layer hidden as expected. 
I'll delve into Layer to find the root cause of this issue -- unless you think there should be a good reason for not letting hidden work on `modal={false}`.

Reply:
Update: 
I pushed a fix for Layer that addresses the issue of hiding the Layer when `position="hidden" && modal={false}` that fixes the issue for SkipLinks, and thanks to this fix I was able to remove this condition and the `tabIndex=0` from the SkipLinks logic.